src/_tests_/App.test.js

```diff
@@ -74,17 +74,6 @@ describe('<App /> integration', () => {
     AppWrapper.unmount();
   });
 
-  test('The state of "numberOfEvents" within App changes when number input changes', async () => {
-    const AppWrapper = mount(<App />);
-    const numberInput = AppWrapper.find(NumberOfEvents).find(
-      '.number-of-events__input',
-    );
-    const eventObject = { target: { value: '15' } };
-    numberInput.at(0).simulate('change', eventObject);
-    expect(AppWrapper.state('numberOfEvents')).toBe('15');
-    AppWrapper.unmount();
-  });
-
   test('changing the number of events', () => {
     const AppWrapper = mount(<App />);
     const EventListWrapper = AppWrapper.find('.EventList');
```

Comment:
The title of the tests implies that the number of events input is going to change, yet the logic inside the tests doesn't change the value of the input. So either add an action where the input gets changed, or change the test title to something like: **returns the maximum of available events when no number gets introduced in the number of events input**

Reply:
Also, `expect(EventListWrapper.find('.EventList')).toHaveLength(1);` is something obvious that doesn't need to be tested within integration tests. You can test however the number of child event elements of the .EventList container.
 `expect(EventListWrapper.find(Event)).toHaveLength(mockData.length);`